app/dashboard/notifications.py

```diff
@@ -406,117 +407,92 @@ def build_github_notification(bounty, event_name, profile_pairs=None):
-    elif event_name == 'rejected_claim':
-        msg = f"{status_header}__The work submission for {natural_value} {bounty.token_name} {usdt_value} " \
-              "has been **rejected** and can now be submitted by someone else.__\n\n * If you would " \
-              f"like to work on this issue you can claim it [here]({absolute_url}).\n * If you've " \
-              f"completed this issue and want to claim the bounty you can do so [here]({absolute_url})\n " \
-              "* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>\n * " \
-              f"${amount_open_work} more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)\n"
+        if bounty.opened_comment:
+            delete_issue_comment(bounty.opened_comment, username, repo)
     elif event_name == 'work_started':
+        msg = {}
         interested = bounty.interested.all().order_by('created')
-        # interested_plural = "s" if interested.count() != 0 else ""
+        interested_context = []
+        started_context = []
+
         from_now = naturaltime(bounty.expires_date)
-        started_work = bounty.interested.filter(pending=False).all()
-        # pending_approval = bounty.interested.filter(pending=True).all()
         bounty_owner_clear = f"@{bounty.bounty_owner_github_username}" if bounty.bounty_owner_github_username else ""
-        approval_required = bounty.permission_type == 'approval'
-
-        if started_work.exists():
-            msg = f"{status_header}__Work has been started__.\n\n"
-        else:
-            msg = f"{status_header}__Workers have applied to start work__.\n\n"
-
-        msg += f"\nThese users each claimed they can complete the work by {from_now}. " \
-               "Please review their questions below:\n\n"
-
-        for i, interest in enumerate(interested, start=1):
-
-            profile_link = f"[{interest.profile.handle}]({interest.profile.url})"
-            action = "started work"
-            if interest.pending:
-                action = 'applied to start work'
-                action += f" _(Funders only: [approve worker]({bounty.approve_worker_url(interest.profile.handle)})"
-                action += f" | [reject worker]({bounty.reject_worker_url(interest.profile.handle)}))_"
-            if not interest.pending and approval_required:
-                action = 'been approved to start work'
-
-            show_dibs = interested.count() > 1 and bounty.project_type == 'traditional'
-            dibs = f" ({get_ordinal_repr(i)} dibs)" if show_dibs else ""
-
-            msg += f"\n{i}. {profile_link} has {action}{dibs}. "
-
-            issue_message = interest.issue_message.strip()
-            if issue_message:
-                msg += f"\t\n * Q: " \
-                       f"{issue_message}"
-            msg += "\n\n"
+        pending_interest = bounty.interested.filter(pending=True).select_related('profile').all()
```

Comment:
Could we dry this up a bit?

Reply:
@mbeacom Tried to do this inside of a function like this instead
`def build_related_profile_pairs(queryset_all):
    # *args:
    #   a 1: QuerySet:  objects to get related profiles for
    queryset_all = queryset_all.select_related('profile').only('profile__handle')
    return queryset_all`